manifests/cluster-manager/hub/cluster-manager-registration-webhook-clusterrole.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
 rules:
 # Allow managedcluster admission to get/list/watch configmaps
 - apiGroups: [""]
-  resources: ["configmaps"]
+  resources: ["configmaps", "pods"]
```

Comment:
why we need this?

Reply:
removed. webhook does not use library-go to create event, so does not need the GET pod rbac.